vmm/src/lib.rs

```diff
@@ -1488,114 +1511,97 @@ impl Vmm {
     }
 
     fn is_instance_initialized(&self) -> bool {
-        let instance_state = {
-            // Use expect() to crash if the other thread poisoned this lock.
-            let shared_info = self.shared_info.read()
-                .expect("Failed to determine if instance is initialized because shared info couldn't be read due to poisoned lock");
-            shared_info.state.clone()
-        };
-        match instance_state {
-            InstanceState::Uninitialized => false,
-            _ => true,
-        }
+        let error_string = "Failed to determine if instance is initialized because shared info couldn't be read due to poisoned lock";
```

Comment:
I think the error message is way too long. Any chance you can also rephrase it a bit?

Reply:
Also, all lines should be less than 100 chars.

Reply:
This is the exact same error string that was returned previously -- I didn't want to change it so that the behaviour of the vmm remains the same (as some users might be used to the errors we already have). But, I've changed it -- and after this, the line length issue is also resolved.